components/contribution-flow/ContributionFlowStepsProgress.js

```diff
@@ -27,16 +27,18 @@ const StepLabel = styled(Span)`
   margin-bottom: 4px;
 `;
 
-const PrettyAmountFromStepDetails = ({ stepDetails, currency, isFreeTier }) => {
+const PrettyAmountFromStepDetails = ({ stepDetails, currency, isFreeTier, isCrypto }) => {
+  const currencySymbol = isCrypto ? stepDetails.currency.value : currency;
```

Comment:
Why not pass directly as `currency`?

Reply:
I've corrected this. I have a feeling this might be an issue when we create an order maybe, but I'll leave that for the other PR. For this one I've added logic to get the crypto currency based on the flow; 

https://github.com/opencollective/opencollective-frontend/blob/b063aaa834459c4f6a1d168de829a29487dbbf79/components/contribution-flow/index.js#L631

That way we can directly pass the `currency`. 👍🏽